spec/features/h2_globus_creation_spec.rb

```diff
@@ -16,7 +16,7 @@
   # may pop up on any attempt to deposit.
   scenario do
     # remove modal for deposit in progress, if present, waiting a bit for some rendering
-    click_link_or_button 'No' if page.has_text?('Continue your deposit')
+    click_link_or_button 'No' if page.has_text?('Continue your deposit', wait: Settings.timeouts.h2_terms_modal_wait)
```

Comment:
I removed these a week or two ago, but I find that I need them to make sure I can get past the "Continue your deposit" modal.